### PR TITLE
Add support for Windows

### PR DESCRIPTION
**Pull request by jkotalik**
Allows for zocket to work on Windows. Code path completely separate from Linux for now as I didn't test Linux locally yet.

### CODE REVIEW

File: src/zocket/Program.cs [hunk 1]
```diff
@@ -68,6 +82,95 @@ private static void Listen(int port, string command)
...
+                _ => throw new ArgumentException(),
+            };
+
+            var currentAssembly = Assembly.GetExecutingAssembly().Location;
+            var reloadIntegrationPath = Path.GetFullPath(Path.Combine(currentAssembly, "..", "ReloadIntegration.dll"));
+
+            using var listenSocket = SetupSocket(port);
+
+            psi.EnvironmentVariables["DOTNET_STARTUP_HOOKS"] = reloadIntegrationPath;
+            psi.EnvironmentVariables["ASPNETCORE_HOSTINGSTARTUPASSEMBLIES"] = "ReloadIntegration";
```
Comment (shirhatti) [inline src/zocket/Program.cs]: Move the shared stuff into a helper and share across both Win/Linux?

Comment (shirhatti) [reply src/zocket/Program.cs]: Also append instead of nuking the variable

**Review by shirhatti — COMMENTED** [review]
Just add a TODO comments :) No need to actually do it

File: src/zocket/Program.cs (hunk 1 above)
Comment (jkotalik) [reply src/zocket/Program.cs]: I'd prefer to move stuff into shared after we stabilize the windows stuff. I can imagine this code path being merged together, removing the need for the helper.

**Pull request merged**